its/src/test/java/com/sonar/it/csharp/UCFGDeserializationTest.java

```diff
@@ -175,7 +167,7 @@ private void runAnalysis(Path projectDir) {
 
   private static ScannerForMSBuild getScannerForMSBuild(Path projectDir) {
     return ScannerForMSBuild.create()
-      .setScannerVersion("4.2.0.1214")
+      .setScannerVersion("4.3.1.1372")
```

Comment:
With this, the logs will no longer suggest to switch the -X or --debug mode thanks to https://jira.sonarsource.com/browse/SQSCANNER-49

However, we still won't be able to see the full stack traces due to https://jira.sonarsource.com/browse/SQSCANNER-50